tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet.md

```diff
@@ -162,7 +162,7 @@ To prepare the parachain collator to be registered on Paseo, follow these steps:
 2. Export the genesis state for the parachain by running the following command:
```

Comment:
```suggestion
2. Export the genesis head for the parachain by running the following command:
```

Reply:
`state` is ok i think, as it is the same in original CLI docs, but i think it's all the same meaning